### PR TITLE
WIP: Multi architecture support

### PR DESCRIPTION
**Pull request by maseabunikie**
Disabled ROS2 components during first tests in RISC-V.

### CODE REVIEW

File: modules/mesh_com/Dockerfile [hunk 1]
```diff
@@ -1,63 +1,24 @@
...
-    rm -rf wpasupplicant_*.deb && \
-    rm -rf /etc/apt/sources.list.d/focal-security.list
-
-ENTRYPOINT [ "/entrypoint.sh" ]
-
 COPY modules/mesh_com/entrypoint.sh /entrypoint.sh
+# Replicate the ROS2 directory structure to so scripts do not need to be refactored.
+RUN mkdir -p /opt/ros/humble/share/bin/
+COPY common/scripts/mesh-11s.sh /opt/ros/humble/share/bin/mesh-11s.sh
+COPY common/scripts/mesh-ibss.sh /opt/ros/humble/share/bin/mesh-ibss.sh
```
Comment (govindsi) [inline modules/mesh_com/Dockerfile:22]: May be we can get rid of ibss now?

**Review by govindsi — COMMENTED** [review]
LGTM as starting steps, this will help in  migrating to rclcpp and nats-c for getting the mesh configuration in phased manner.

**Review by govindsi — APPROVED** [review]

**Review by joenpera — APPROVED** [review]

File: modules/mesh_com/Dockerfile (hunk 1 above)
Comment (joenpera) [reply modules/mesh_com/Dockerfile:22]: True. We don't need ibss script into a device anymore.

**Comment by maseabunikie** [comment]
I will remove the ibss script.
There are some issues and I cannot get the built container to work.

**Comment by maseabunikie** [comment]
Closing because not going to be merged yet into develop.

**Pull request closed without merging**